eng/harvest.targets

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <PropertyGroup>
+    <HarvestProject>$(RepoRoot)\src\harvest\harvest.csproj</HarvestProject>
+    <SkipCopyBuildProduct>true</SkipCopyBuildProduct>
+    <TargetsForTfmSpecificContentInPackage>$(TargetsForTfmSpecificBuildOutput);AddHarvestedFilesToPackage</TargetsForTfmSpecificContentInPackage>
+  </PropertyGroup>
+
+  <!-- Override CoreCompile so that we don't build anything. -->
+  <Target Name="CoreCompile" />
+  <!-- Since we are't building, make sure the outputgroups used by packaging will be empty -->
+  <Target Name="BuiltProjectOutputGroup">
+    <ItemGroup>
+      <BuiltProjectOutputGroupOutput Remove="@(BuiltProjectOutputGroupOutput)" />
+    </ItemGroup>
+  </Target>
+  <Target Name="DebugSymbolsProjectOutputGroup">
+    <ItemGroup>
+      <!-- make sure to delete the static-defined item -->
+      <DebugSymbolsProjectOutputGroupOutput Remove="@(DebugSymbolsProjectOutputGroupOutput)" />
+    </ItemGroup>
+  </Target>
+
+  <Target Name="_GetPathToHarvest">
+    <MSBuild Projects="$(HarvestProject)"
+             Targets="GetPathToHarvest"
+             Properties="PackageToHarvest=$(PackageId)">
+      <Output TaskParameter="TargetOutputs" PropertyName="_pathToHarvest"/>
+    </MSBuild>
+  </Target>
+
+  <Target Name="AddHarvestedFilesToPackage"
+          DependsOnTargets="_GetPathToHarvest">
+    <ItemGroup>
+      <_harvestedItems Include="$(_pathToHarvest)\lib\$(TargetFramework)\*.*" PackagePath="lib\$(TargetFramework)" />
```

Comment:
Nit: We generally use `_HarvestedItems` capitalization for "private" properties/items/targets.